rust/src/transformations/make_stable_expr/expr_sum/mod.rs

```diff
@@ -125,6 +125,11 @@ where
         _ => TI::Sum::inf_cast(*d_in)?.inf_mul(&l.alerting_abs()?.total_max(u)?),
     };
 
+    // 'mnp_check: this invariant is used later
```

Comment:
I'm expecting to see `'mnp_check` in the code here, but it's just in an error message. Is that right?

Reply:
yep, the error message should never be seen by the user, and it references another location by a comment.